scripts/lib/mkcloud-onhost.sh

```diff
@@ -54,10 +54,9 @@ done
 iptables_unique_rule PREROUTING -t nat -p tcp --dport \$(( $cloud_port_offset + 6080 )) \\
     -j DNAT --to-destination $net_public.2
 
-# need to delete+insert on top to make sure our ACCEPT comes before libvirt's REJECT
-for x in D I ; do
-    iptables -\$x FORWARD -d $net_admin.0/24 -j ACCEPT
-    iptables -\$x FORWARD -d $net_public.0/24 -j ACCEPT
+# delete libvirt's REJECT rules to allow public net
```

Comment:
I followed https://www.ibm.com/support/knowledgecenter/en/linuxonibm/liaat/liaatsecnetfilt.htm and tried adding below <interface type='network'>
```
<filterref filter='allow-ipv4'/>
```
and tried
virsh nwfilter-edit clean-traffic
to clean the reject rules

but that did not get rid of the default reject rules. Any further hints highly welcome.